integration/runner/runner.go

```diff
@@ -117,11 +118,17 @@ func PushAndRunTests(host, testDir string) error {
 	// Run the tests.
 	glog.Infof("Running integration tests targeting %q...", host)
 	err = RunCommand("godep", "go", "test", "github.com/google/cadvisor/integration/tests/...", "--host", host, "--port", portStr)
```

Comment:
nit: To avoid having two commands that are expected to be identical, can we increase the retry count to be `6` instead and run the command only inside the loop or cache the command in a separate var?